lib/ASTGen/Sources/ASTGen/Bridge.swift

```diff
@@ -167,10 +167,14 @@ extension TokenSyntax {
   /// - Parameter astgen: The visitor providing the `ASTContext`.
   @inline(__always)
   func bridgedIdentifier(in astgen: ASTGenVisitor) -> BridgedIdentifier {
-    var text = self.text
-    return text.withBridgedString { bridged in
-      astgen.ctx.getIdentifier(bridged)
+    var text = self.rawText
+    if rawText == "_" {
+      return nil
+    }
+    if rawText.count > 2 && rawText.hasPrefix("`") && rawText.hasSuffix("`") {
+      text = .init(rebasing: text.dropFirst().dropLast())
```

Comment:
I guess SwiftSyntax doesn't have a better way of doing this?

Reply:
Like ``.trim("`")``? No.
This is basically just slicing `RandomAccessCollection`, and I think this is the simplest.

Reply:
I just meant a utility for getting the actual identifier for an escaped identifier (e.g we have `getText` vs `getRawText` on Token in the old parser)